cmd/bbolt/main_test.go

```diff
@@ -4,7 +4,9 @@ import (
 	"bytes"
 	crypto "crypto/rand"
 	"encoding/binary"
+	"encoding/hex"
 	"fmt"
+	"go.etcd.io/bbolt/internal/guts_cli"
```

Comment:
Need to put all `go.etcd.io/xxx` together.

Reply:
done 👍🏽